Assets/Scripts/Models/Buildable/Furniture.cs

```diff
@@ -368,6 +368,18 @@ public bool OnlyUseDefaultSpriteName
             return !string.IsNullOrEmpty(getSpriteNameAction);
         }
     }
+
+    /// <summary>
+    /// Whether the furniture has power or not.
+    /// </summary>
+    /// <returns>True if the furniture has power.</returns>
+    public bool HasPower
+    {
+        get
+        {
+            return PowerConnection == null || World.Current.PowerNetwork.HasPower(PowerConnection);
```

Comment:
` return PowerConnection == null`
do you intend for objects who have no connection to return that they do have power? would they by definition have no power?